docs/getting_started.md

```diff
@@ -5,6 +5,9 @@
 
 1. Python 3.7
 2. AWS Account credentials when testing with AWS platforms
+   check default section in ~/.aws/credentials for access/secret key.
+3. OC client binary is installed on your localhost in /usr/local/bin or
+   /usr/bin(running oc version on terminal should display version > 3.11).
```

Comment:
You mean to have `oc` placed somewhere in `PATH`. Also you may want to link to https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/